### PR TITLE
test: wait for frontend rollout before getting app logs

### PR DESCRIPTION
**Pull request by kargakis**
Fixes https://github.com/openshift/origin/issues/11682

@mfojtik ptal

### CODE REVIEW

**Comment by kargakis** [comment]
[test]

File: test/end-to-end/core.sh [hunk 1]
```diff
@@ -419,6 +419,8 @@ frontend_pod=$(oc get pod -l deploymentconfig=frontend --template='{{(index .ite
 os::cmd::expect_success_and_text "oc exec -p ${frontend_pod} id" '1000'
 os::cmd::expect_success_and_text "oc rsh pod/${frontend_pod} id -u" '1000'
 os::cmd::expect_success_and_text "oc rsh -T ${frontend_pod} id -u" '1000'
+# Wait for the rollout to finish
+os::cmd::expect_success "oc rollout status dc/frontend --revision=1"
```
Comment (liggitt) [inline test/end-to-end/core.sh]: are we sure this will always be 1? I thought there was a race with it being either 1 or 2

**Comment by liggitt** [comment]
and if there's a race issue with the helper, then we should use `rollout status` everywhere, right?

File: test/end-to-end/core.sh (hunk 1 above)
Comment (kargakis) [inline test/end-to-end/core.sh]: The race was for generation/observedGeneration, --revision is latestVersion which is always 1 (nothing else starts a second deployment)

**Comment by kargakis** [comment]
> and if there's a race issue with the helper, then we should use rollout status everywhere, right?

Yes, that is a different issue. I am fixing the one I linked in the description

File: test/end-to-end/core.sh (hunk 1 above)
Comment (liggitt) [inline test/end-to-end/core.sh]: ah. can we just leave it empty to use the default of the last revision?

**Comment by kargakis** [comment]
https://github.com/openshift/origin/issues/11728 [test]

**Comment by mfojtik** [comment]
LGTM [merge]

**Pull request merged**